source/braille.py

```diff
@@ -3841,3 +3845,18 @@ def _speakOnRouting(info: textInfos.TextInfo):
 
 	info.expand(textInfos.UNIT_CHARACTER)
 	spellTextInfo(info)
+
+
+def _speakOnNavigatingByUnit(info: textInfos.TextInfo):
```

Comment:
If this function is supposed to speak the reading unit, I think I'd do the expansion in here rather than in the methods that call it. May be even do the copying in here to be on the safe side of things.